src/components/ExploreBlogs/index.jsx

```diff
@@ -2,79 +2,67 @@ import React from "react";
 import "./index.css";
 import { Row, Container, Col, Button } from "react-bootstrap";
 import { useTranslation } from "react-i18next";
-import image from "./images/placeHolderImage.jpg";
+import { Link } from "react-router-dom";
 
 // Explore-Blogs section shows the last 3 blog posts in the database.
 // If you click on a single blog post, you go to that blog's page
 // If you click on show more button, you go to blogs list page
 const ExploreBlogs = () => {
   const { t } = useTranslation();
 
+  const [posts, setPosts] = React.useState([]);
+  const _URL =
+    "https://public-api.wordpress.com/wp/v2/sites/nobarriers650349093.wordpress.com/posts";
+
+  React.useEffect(() => {
+    const fetchPosts = async () => {
+      const data = await fetch(_URL);
+      const fetchedPosts = await data.json();
+      setPosts(fetchedPosts.slice(0, 3));
```

Comment:
Can you change the magic numbers?

Reply:
why changing them tho?
I sliced them in order to render only 3 posts in the homePage.

Reply:
@khaldoon9kh yes change these into consts there shouldn't be magic numbers in the code

Reply:
Resolved